src/integrationTest/java/org/opensearch/test/framework/TestSecurityConfig.java

```diff
@@ -967,12 +967,7 @@ public void updateInternalUsersConfiguration(Client client, List<User> users) {
     }
 
     static String hash(final char[] clearTextPassword) {
-        final byte[] salt = new byte[16];
-        new SecureRandom().nextBytes(salt);
-        final String hash = OpenBSDBCrypt.generate((Objects.requireNonNull(clearTextPassword)), salt, 12);
-        Arrays.fill(salt, (byte) 0);
-        Arrays.fill(clearTextPassword, '\0');
-        return hash;
+        return passwordHasher.hash((Objects.requireNonNull(clearTextPassword)));
```

Comment:
No need to use `Objects.requireNonNull` here AFAIU `passwordHasher.hash` already checks on the null value and throws `OpenSearchException`.

Reply:
Done!